src/Controller/Component/AuthComponent.php

```diff
@@ -597,7 +597,7 @@ public function deny($actions = null)
     public function setUser(array $user)
     {
         $this->session->renew();
-        $this->session->write($this->sessionKey, $user);
+        $this->session->write($this->sessionKey . '.user', $user);
```

Comment:
i changed it because all other keys are lowercase.
you should use Auth->user() to access user data